README.md

```diff
@@ -1,157 +1,179 @@
+    p.instrument(:LibName)
     ...
 end
 ```
 
-Valid lib names are listed below with the instrumentation documentation.
-
-`configure` accepts several optional parameters:
-- `tracer`: a preconfigured OpenTracing tracer to use. If one is not provided,
-  a new tracer will be initialized.
-  - Default: `nil`
-- `ingest_url`: this is the Smart Agent or Smart Gateway endpoint to which spans are sent by the tracer.
-  - Default: `http://localhost:9080/v1/trace`
-- `service_name`: service name to send spans under.
-  - Default: `signalfx-ruby-tracing`
-- `access_token`: SignalFx access token for authentication.  Unnecessary for most deployments.
-  - Default: `''`
-
-Environment variables can be used to configure `service_name` and `access_token`
-if not given to the `configure` method.
-
-```bash
-export SIGNALFX_SERVICE_NAME="<service_name>"
-export SIGNALFX_ENDPOINT_URL="<url>"
-export SIGNALFX_ACCESS_TOKEN="<token>"
-```
-
-If these environment variables are not set, the values will default to the ones
-listed above.
+## Usage information for each library
 
-# Instrumentation
-
-This section contains details and configuration for specific frameworks.
-
-### Runtimes
-
-- MRI Ruby (CRuby) 2.0+
-
-### Web servers
-
-- Puma >= 3.0.0
-- Passenger >= 5.0.25
-
-Instrumentation for routes using these web servers is provided through Rack.
-If using a framework that builds on top of Rack, such as Rails or Sinatra, our
-instrumentation includes Rack instrumentation. In these cases, the routes
-through the web server will be automatically traced.
-
-When interfacing with these web servers as a Rack application, please configure
-[Rack instrumentation](#rack) and insert it as middleware.
+Here's information about instrumenting each supported library.
 
-### Libraries/Frameworks
-
-| Library                             | Versions Supported |
-| ----------------------------------- | ------------------ |
-| [ActiveRecord](#active-record)      | ~> 5.0             |
-| [Elasticsearch](#elasticsearch)     | >= 6.0.2           |
-| [Faraday](#faraday)                 | >= 0.9.0           |
-| [Grape](#grape)                     | >= 0.13.0          |
-| [Mongo](#mongo)                     | >= 2.1.0           |
-| [Mysql2](#mysql2)                   | >= 0.4.0           |
-| [Net::HTTP](#nethttp)               | Ruby >= 2.0        |
-| [Pg](#pg)                           | >= 0.18.0          |
-| [Rack](#rack)                       | >= 0.1             |
-| [Rails](#rails)                     | >= 3.0.0           |
-| [Redis](#redis)                     | >= 4.0.0           |
-| [RestClient](#restclient)           | >= 1.5.0           |
-| [Sequel](#sequel)                   | >= 3.47.0          |
-| [Sidekiq](#sidekiq)                 | >= 0.7.0           |
-| [Sinatra](#sinatra)                 | >= 1.0.0           |
-
-## Active Record
+### Active Record
 
 This instrumentation creates spans for each Active Record query using the Active
```

Comment:
Do you mind updating to state that active record instrumentation is already provided by the rails instrumentation, so is not necessary if using that?